docs/sources/administration/plugin-management/index.md

```diff
@@ -144,11 +155,13 @@ If you are logged in to Grafana Cloud when you add a plugin, log out and back in
 
 ### Install plugin on local Grafana
 
-Follow the instructions on the Install tab. You can either install the plugin with a Grafana CLI command or by downloading and uncompress a .zip file into the Grafana plugins directory. We recommend using Grafana CLI in most instances. The .zip option is available if your Grafana server does not have access to the internet.
+Follow the instructions on the Install tab. You can either install the plugin with a Grafana CLI command or by downloading and uncompress a `.zip` file into the Grafana plugins directory. We recommend using Grafana CLI in most instances. The `.zip` option is available if your Grafana server does not have access to the internet.
```

Comment:
The challenge I have is that we have just described the primary installation method, the catalog. 

As minor editorial changes, these are fine, but I feel they are not improving the page - as we should be thinking about whether the sections are required or should be reframed.

For Cloud, given the default is now that plugin installation is the same as on premise, and we have Grafana Cloud docs, this section feels pointless within this page and is more a "detail" for the cloud docs perhaps. The for local, instead this could be about "Installing plugins using the Grafana CLI"

Reply:
Good Point.

I suggest the following. We keep the heading https://grafana.com/docs/grafana/latest/administration/plugin-management/#install-plugin-on-local-grafana

And then mention that users can use the Plugin Catalog as it is available by default. And if want to install via CLI then refer them to the https://grafana.com/docs/grafana/latest/cli/ which itself is very detailed (I can move the unzip example there so that we do not loose it).

For the cloud do you know the page which we can refer to?